agr_variant_indexer/src/main/java/org/alliancegenome/indexer/variant/es/managers/SourceDocumentCreation.java

```diff
@@ -433,7 +433,7 @@ public void run() {
         private final ObjectMapper mapper = new ObjectMapper();
 
         public void run() {
-            mapper.setSerializationInclusion(JsonInclude.Include.NON_NULL);
+            //mapper.setSerializationInclusion(JsonInclude.Include.NON_NULL);
```

Comment:
Why are you commenting this out?

Reply:
`        /**
         * Value that indicates that only properties with non-null
         * values are to be included.
         */
        NON_NULL,`
        
`         * With 2.7, definition has been tightened back to only containing types explained
         * above (null, absent, empty String, empty containers), and now
         * extended definition may be specified using {@link #NON_DEFAULT}.
         */
        NON_EMPTY,`

Reply:
you don't need both since we have NON_EMPTY we don't need NON_NULL. And I wanted to make sure they were not conflicting with each other.